### ISSUE TITLE
Windows book includes attribute markup for user-supplied info, AsciiDoctor complains

### ISSUE THREAD

**Issue by kowen-rh** [red-hat-openjdk/docs#4]
AsciiDoctor complains while rendering the `modules/proc-windows-installing-openjdk-zip.adoc` procedure that the `your_install_path` attribute does not exist due to the use of `${YOUR_INSTALL_PATH}` in the source.

Use of `${YOUR_INSTALL_PATH}` occurs on lines 20, 53, and 58 of `modules/proc-windows-installing-openjdk-zip.adoc`.

We can remove the warnings by using `<YOUR_INSTALL_PATH>` or using a passthrough macro (`pass:[${YOUR_INSTALL_PATH}]`).

**Comment by kowen-rh** [comment]
I'm trying to fix this one and I've run into a problem I _know_ I've seen before, but can't seem to quite fix. Namely, the _`<YOUR_INSTALL_PATH>`_ text in this source block should render with italics, but doesn't:

```adoc
[subs="+quotes,attributes"]
----
C:\> setx /m PATH "%PATH%;_<YOUR_INSTALL_PATH>_\bin"
----
```

I'm not sure why this isn't working as I expected. :confused:

**Comment by kowen-rh** [comment]
Got it figured out, clearly -- we needed to use two underscores instead of one due to the semicolon. :)

**Comment by kowen-rh** [comment]
Fixed by #7.
